models/User.js

```diff
@@ -26,10 +26,9 @@ const UserSchema = new Schema({
 		unique: true,
 		dropDups: true
 	},
-	isAdmin: {
```

Comment:
@neilbrub will removing this break anything?

Reply:
@picklechips Removing `isAdmin` should be fine as long as client-side implementations are changed in parallel (looks all good in the "Added Don information!" PR in the mobile repo - need to make sure the PRs are merged simultaneously).
A potential issue I see here is that user objects created  before this update might not have the `role` field, and if so invoking `user.role.includes(...)` would raise a TypeError. I'll bring this up client-side though.

Reply:
@AngelaKrone not real sure how expo works but can we rely on everyone's app being updated?